src/cljs/athens/views/blocks/content.cljs

```diff
@@ -335,19 +334,18 @@
                         "1em")]
         [:div {:class "block-content" :style {:font-size font-size}}
          ;; NOTE: komponentit forces reflow, likely a performance bottle neck
-         [autosize/textarea {:value          (:string/local @state)
-                             :class          ["textarea" (when (and (empty? @selected-items) @editing?) "is-editing")]
-                             ;; :auto-focus     true
-                             :id             (str "editable-uid-" uid)
-                             :on-change      (fn [e] (textarea-change e uid state))
-                             :on-paste       (fn [e] (textarea-paste e uid state))
-                             :on-key-down    (fn [e] (textarea-keydown/textarea-key-down e uid state))
-                             :on-blur        (fn [_] (db/transact-state-for-uid (or original-uid uid) state))
-                             :on-click       (fn [e] (textarea-click e uid state))
-                             :on-mouse-enter (fn [e] (textarea-mouse-enter e uid state))
-                             :on-mouse-down  (fn [e] (textarea-mouse-down e uid state))}]
+         (when (or (:render-editable @state) editing?)
```

Comment:
Clever

Reply:
Wait can you actually explain this a little? And what is happening on `mouse-enter` and `mouse-leave`?

Reply:
I've renamed this var and added some comments around where it's used.